### DISCUSSION TITLE
AgentTesla.LQL!MTB virus detected

### DISCUSSION THREAD

**Discussion by NiceShot** [miroslavpejic85/p2p#52] [General]
When I tried to compiled the code, windows defender detected this threat: AgentTesla.LQL!MTB

**Comment by miroslavpejic85** [comment]
https://github.com/miroslavpejic85/p2p#note

<img width="880" alt="readme-note" src="https://user-images.githubusercontent.com/70838434/230895609-1c8c7b25-3462-4f54-bf2b-c1e0b272c177.png">
